ingestion_tools/scripts/data_validation/fixtures/data.py

```diff
@@ -44,9 +123,9 @@ def get_headers(mrcfiles: List[str]) -> Dict[str, MrcInterpreter]:
 
 
```

Comment:
from here down to the next comment, it's just refactoring the code knowing that there will only be one mrc file for these entities